app/src/pages/inside/historyPage/historyView/historyTable/utils.js

```diff
@@ -0,0 +1,21 @@
+/*
+ * Copyright 2019 EPAM Systems
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+export const calculateMaxRowItemsCount = (history) =>
```

Comment:
Maybe `getMaxRowItemsCount` would be shorter and better?

Reply:
It calculates the value according to the arguments.
It isn't part of any instance and doesn't use properties defined in its scope.
I think this is not critical in this case.

Reply:
Ok